handsontable/src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -84,6 +85,33 @@ const COLUMN_SIZE_MAP_NAME = 'autoColumnSize';
  *   // code...
  * }
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * // First, let's contruct Handsontable
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ *   autoColumnSize={true}
+ * />
+ *
+ * ...
+ *
+ * // Access to plugin instance:
+ * const plugin = hotRef.getPlugin('autoColumnSize');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.